### ISSUE TITLE
Pygame shows empty display

### ISSUE THREAD

**Issue by jasonxyliu** [h2r/pomdp-py#5]
Hi,

I was running the MOS example at [https://h2r.github.io/pomdp-py/html/examples.mos.html](url) by 
`$ python -m pomdp_problems.multi_object_search.problem`

A pygame window is displayed, but it remains empty during the running of the program.
The console output indicates that the planning is progressing.

Does anyone have any insights of how to fix this bug? Thanks in advance!

**Version**
MacOS 10.15.5 (19F101)

```
$ conda list
# packages in environment at /Users/UserName/anaconda/envs/ltl_pomdp:
#
# Name                    Version                   Build  Channel
ca-certificates           2020.6.24                     0  
certifi                   2020.6.20                py37_0  
cycler                    0.10.0                    <pip>
cython                    0.29.21          py37hb1e8313_0  
decorator                 4.4.2                     <pip>
kiwisolver                1.2.0                     <pip>
libcxx                    10.0.0                        1  
libedit                   3.1.20191231         h1de35cc_1  
libffi                    3.3                  hb1e8313_2  
matplotlib                3.3.0                     <pip>
ncurses                   6.2                  h0a44026_1  
networkx                  2.4                       <pip>
numpy                     1.19.1                    <pip>
opencv-python             4.4.0.40                  <pip>
openssl                   1.1.1g               h1de35cc_0  
Pillow                    7.2.0                     <pip>
pip                       20.2.2                   py37_0  
pomdp-py                  1.2                       <pip>
pygame                    1.9.6                     <pip>
pygraphviz                1.6                       <pip>
pyparsing                 2.4.7                     <pip>
python                    3.7.7                hf48f09d_4  
python-dateutil           2.8.1                     <pip>
readline                  8.0                  h1de35cc_0  
scipy                     1.5.2                     <pip>
setuptools                49.2.1                   py37_0  
six                       1.15.0                    <pip>
sqlite                    3.32.3               hffcf06c_0  
tk                        8.6.10               hb0a8c7a_0  
wheel                     0.34.2                   py37_0  
xz                        5.2.5                h1de35cc_0  
zlib                      1.2.11               h1de35cc_3
```

**Partial Console output**
```
==== Step 112 ====
Action: move-xyth-North
Observation: MosOOObservation({})
Reward: -2
Reward (Cumulative): 2796
Find Actions Count: 3
__num_sims__: 225
==== Step 113 ====
Action: move-xyth-South
Observation: MosOOObservation({})
Reward: -2
Reward (Cumulative): 2794
Find Actions Count: 3
__num_sims__: 229
==== Step 114 ====
Action: move-xyth-West
Observation: MosOOObservation({})
Reward: -2
Reward (Cumulative): 2792
Find Actions Count: 3
__num_sims__: 230
==== Step 115 ====
Action: move-xyth-West
Observation: MosOOObservation({})
Reward: -2
Reward (Cumulative): 2790
Find Actions Count: 3
__num_sims__: 216
==== Step 116 ====
Action: move-xyth-North
Observation: MosOOObservation({})
Reward: -2
Reward (Cumulative): 2788
Find Actions Count: 3
__num_sims__: 226
==== Step 117 ====
Action: move-xyth-East
Observation: MosOOObservation({})
Reward: -2
Reward (Cumulative): 2786
Find Actions Count: 3
__num_sims__: 245
==== Step 118 ====
Action: move-xyth-South
Observation: MosOOObservation({})
Reward: -2
Reward (Cumulative): 2784
Find Actions Count: 3
__num_sims__: 248
```

**Comment by zkytony** [comment]
I think this is the same issue Monica was dealing with on her Mac. The fix is to upgrade your pygame package to 0.20.0+.

See if that works!

**Comment by jasonxyliu** [comment]
Upgrading pygame fixed the bug. Thanks!
If you use virtualenv, run `python3 -m pip install -U pygame==2.0.0.dev6 --user` as suggested by this link [https://www.pygame.org/wiki/GettingStarted#Mac%20installation](url)
In conda environment, run `pip install -U pygame==2.0.0.dev6`
